sm-core-model/src/main/java/com/salesmanager/core/model/customer/Customer.java

```diff
@@ -147,7 +149,14 @@
 		
 	})
 	private List<Group> groups = new ArrayList<Group>();
-	
+
+	@Column(name = "CUTOMER_IMAGE")
+	private String customerImage;
+
+	@JsonIgnore
+	@Transient
+	private MultipartFile image = null;
```

Comment:
why you put a multipartFile object here? this is an entity class!

Reply:
Because they using the entity as the jsp form model and I needed to capture the file in the controller